command/title_body_survey.go

```diff
@@ -13,8 +13,16 @@ import (
 )
 
 type Action int
+type metadataStateType int
+
+const (
+	issueMetadata metadataStateType = iota
+	prMetadata
+)
 
 type issueMetadataState struct {
```

Comment:
I feel that the name of this struct should be changed, since it is not only used for issues, but for PRs also.  
Possibly to `metadataState`? :thinking:

Reply:
In github-y terms, PRs _are_ issues since they are a subset of Issues. If we renamed this to `metadataState` in the `command` package then it might not be clear what the "metadata" applies to. I think the only more precise name would be "issueOrPullRequestMetadataState`

Reply:
> In github-y terms, PRs are issues since they are a subset of Issues

Oh. I didn't know that.

> I think the only more precise name would be "issueOrPullRequestMetadataState`

Agreed. Can we do that, just for extra clarity?